IreneBranch/Project1

```diff
@@ -0,0 +1 @@
+Subproject commit b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996
```

Comment:
done! (I think)

On 2 May 2016 at 19:22, Ariadni-Karolina Alexiou <notifications@github.com>
wrote:

> In IreneBranch/Project1
> <https://github.com/IreneG5/Project1/pull/2#discussion_r61779610>:
>
> > @@ -0,0 +1 @@
> > +Subproject commit b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996
>
> git rm this folder from your branch
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/IreneG5/Project1/pull/2/files/ea58a2dc7f0a59d8ae8da51f7d79e2a432cab3a1#r61779610>
>